public/template/layout.jade

```diff
@@ -8,15 +8,15 @@ html(lang="en")
 
     if config.noCDN
 
-      link(href="/stylesheet/bootstrap.min.css", rel="stylesheet")
-      link(href="/stylesheet/bootstrap-responsive.min.css", rel="stylesheet")
+      link(href="/stylesheet/bootstrap-combined.no-icons.min.css", rel="stylesheet")
```

Comment:
Is the same CDN but local: https://maxcdn.bootstrapcdn.com/twitter-bootstrap/2.3.2/css/bootstrap-combined.no-icons.min.css